app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+
+  def create
+    @job = Job.find(params[:job_id])
+    @comment = @job.comments.new(comment_params)
+
+    if @comment.save
+      flash[:success] = "Comment saved!"
+      redirect_to company_job_path(@job.company, @comment.job)
```

Comment:
why not use `company-job_path(@job.company, @job)` adding `@comment.job` can get confusing